backend/src/extension.ts

```diff
@@ -7,17 +7,15 @@ import {RpcExtension} from '@sap-devx/webview-rpc/out.ext/rpc-extension';
 import { YouiLog } from "./youi-log";
 import { OutputChannelLog } from './output-channel-log';
 import { GeneratorFilter } from './filter';
+import backendMessages from "./messages";
```

Comment:
couldn't we implement it with i18n?

Reply:
I have found vue-i18n lib, but it only helps to translate strings, but does not help to replace strings. 
According to Avital in case we need translated strings it will happen only in 2023 :)

Reply:
ok